pydash/arrays.py

```diff
@@ -1435,14 +1477,64 @@ def union(*arrays):
 
     Example:
 
-        >>> union([1, 2, 3], [2, 3, 4], [3, 4, 5])
+        >>> union([[1, 2, 3], [2, 3, 4], [3, 4, 5]])
         [1, 2, 3, 4, 5]
 
     .. versionadded:: 1.0.0
     """
     return uniq(flatten(arrays))
 
 
+def union_by(arrays, callback=None):
```

Comment:
Let's mirror `union` by having `*arrays` as the argument instead of only `arrays`.